file-formats/apob/type/zif_aff_apob_v1.intf.abap

```diff
@@ -0,0 +1,62 @@
+INTERFACE zif_aff_apob_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Application Object Description Type</p>
+    "! Attributes of Application Object Description Type
+    BEGIN OF ty_apob_text,
+      "! <p class="shorttext">Language Code</p>
+      "! Language Code
+      "! $required
+      lang    TYPE c LENGTH 1,
```

Comment:
We suggest not to use abbreviations, so I recommend using 
```suggestion
      language_code    TYPE c LENGTH 1,
```

Reply:
changed